CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 ## 1.2.0-SNAPSHOT
 
 * Fix help-for for some restructure methods [#275](https://github.com/metosin/compojure-api/pull/275) by [Nicolás Berger](https://github.com/nberger)
+* In `compojure.api.swagger`, the `swagger-ui` and `swagger-docs` now take options map with `uri` key instead of separate optional path & vararg opts.
```

Comment:
actually, the function arity has changed and the routes are invoked only at runtime -> without turning those into macros, there will be a runtime error. But most likely people see that immedeately, so should fail nice'ish. And these are not in the `sweet` api I guess 99% people are using.